srcpkgs/eza/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'eza'
 pkgname=eza
-version=0.13.0
+version=0.13.1
 revision=1
 build_style=cargo
 hostmakedepends="pandoc pkg-config"
```

Comment:
pandoc is nocross which makes this package nocross too with our builders, could we have something like

```
hostmakedepends="$(vopt_if man pandoc)"

build_options="man"
desc_option_man="Use pandoc for manpages"

case "$XBPS_MACHINE" in
	x86_64*|i686) build_options_default="man" ;;
esac

post_install() {
	if [ "$build_option_man" ]; then
		...
	fi
}
```

Reply:
I notified upstream in https://github.com/eza-community/eza/issues/357 that prerendered manpages are desired, I assumed eza was merged as-is because this is not a huge problem and can wait until we eliminate pandoc from this template altogether.

Reply:
> pandoc is nocross which makes this package nocross too with our builders

but it's in hostmakedepends